docs/source/Fabric-FAQ.rst

```diff
@@ -56,7 +56,10 @@ Security & Access Control
   Do the orderers see the transaction data?
 
 :Answer:
-  No, the orderers only order transactions, they do not open the transactions.
+  Orderers receive endorsements from the client. The endorsement payload has
```

Comment:
Well, if you would like to expand with more details I think it has to be more accurate and reflect the actual flow, for example:

Orderers receive a transaction, which includes endorsements from the peers collected by the client, moreover endorsement payload contains not only RWSet, but also signatures of the endorsing peers over the hashed value of the simulation results including endorsement proposal. 

That being said, I do not see real issue with previous answer which is very concrete and precisely answering the question.

Reply:
I found this to be misleading because the community misinterpreted "does orderers read" to be "does orderers have access". This misinterpretation leading to designs where privacy is assumed. There can either be another question on the flow or this question can be clarified with additional details.

Let me know your thoughts.

Reply:
Thanks Dave for the feedback. I take it as a yes to providing clarification on this answer itself. Edited the wordings as per suggestion.

Reply:
> I found this to be misleading because the community misinterpreted "do orderers read?" to be "do orderers have access?". This misinterpretation could lead to designs where privacy is assumed. Wordings here carry more weightage because this question is under FAQ for data privacy. There can either be another question on the flow or this question can be clarified with additional details.
> 
> Let me know your thoughts.

I do agree clarification is needed, just thought that it could be a bit more accurate, and Dave's suggestion is actually a good one.